### DISCUSSION TITLE
Can tif to pdf convert handle a tif format file that has total pages set to 0?

### DISCUSSION THREAD

**Discussion by pbogosian-ccs** [ImageMagick/ImageMagick#6812] [Help]
### ImageMagick version

6.9.10-68

### Operating system, version and so on

CentOS 7

### Description

I have a 251 page file that is causing a core dump when I use ImageMagick to convert it to PDF (also reports out of memory). It use to work, and the only difference I see in this tif file is that total pages is set to 0 (from what I read, this is allowed in the tif specification). I'm using ImageMagick to convert the pages from 204x98 dpi to 204x196 dpi. The tif file 's first page Pagenumber is 0,0 and the second page number is 1,0 etc.

Any insight on this would be appreciated. Thank you, Philip

**Comment by snibgo** [comment]
I suggest you upgrade to the current v6 version (v6.9.12-98). Even better, to the current v7 version. Does that fix the core dump?

IM should never core dump. The developers may need to see the file, to fix the problem.
